themes/beaver/layouts/_default/baseof.html

```diff
@@ -1,6 +1,75 @@
-{{ partial "page/header.html" . }}
-{{ block "main" . }}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta name="color-scheme" content="light dark">
+<link rel="icon" type="image/png" href="data:image/png;base64,">
+<title>{{ .Title }}</title>
+<meta name="description" content="{{ .Params.description }}" />
+{{ if and (eq .Kind "taxonomy") (eq .Section "tags") }}
+{{ $canonical := .Permalink | replace "/tags/" "/blog/tags/" }}
+<link rel="canonical" href="{{ $canonical }}" />
+{{ else }}
+<link rel="canonical" href="{{ .Permalink }}" />
 {{ end }}
-{{ block "footer" . }}
+
+<link rel="icon" href="/wp-content/uploads/2022/09/cropped-jet-thoughts-logo-32x32.png" sizes="32x32" />
+<link rel="icon" href="/wp-content/uploads/2022/09/cropped-jet-thoughts-logo-192x192.png" sizes="192x192" />
+<link rel="apple-touch-icon" href="/wp-content/uploads/2022/09/cropped-jet-thoughts-logo-180x180.png" />

```

Comment:
@AucT  please schedule an issue to make it as assets

Reply:
++